### PR TITLE
717 - Make Cover Letter file name not use UNIX time

### PR DESCRIPTION
**Pull request by FrancescoVassalli**
address #717

### CODE REVIEW

**Review by surapuramakhil — CHANGES_REQUESTED** [review]
@FrancescoVassalli request to address this changes.

**Comment by cjbbb** [comment]
When submitting a PR, you can also add a more detailed description of the changes to help us locate them quickly.

File: src/ai_hawk/linkedIn_easy_applier.py [hunk 1]
```diff
@@ -466,8 +470,11 @@ def _create_and_upload_resume(self, element, job):
 
         while True:
             try:
-                timestamp = int(time.time())
-                file_path_pdf = os.path.join(folder_path, f"CV_{timestamp}.pdf")
+                file_path_pdf = os.path.join(folder_path, f"CV_{job.company}.pdf")
```
[the remark this answers is not in the document]
Comment (FrancescoVassalli) [reply src/ai_hawk/linkedIn_easy_applier.py:473]: The candidate name does not seem to be easily accessible here

**Comment by sarob** [comment]
**not sarcastic** is this a priority to spend time on?

File: src/ai_hawk/linkedIn_easy_applier.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (sarob) [reply src/ai_hawk/linkedIn_easy_applier.py:473]: including candidate name seems redundant doesn't it?

**Comment by FrancescoVassalli** [comment]
> **not sarcastic** is this a priority to spend time on?

I think the UNIX time in the filename is important to take out because anti-bot operations could look for that

File: src/ai_hawk/linkedIn_easy_applier.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (surapuramakhil) [reply src/ai_hawk/linkedIn_easy_applier.py:473]: it helps hiring companies. CV with candidate name is for uploading in job portal. They do get a bunch of CV files. more professional way

for tracking / logs of Job applicants it should be CV with job ID/details like company & position

**Comment by surapuramakhil** [comment]
> > **not sarcastic** is this a priority to spend time on?
> 
> I think the UNIX time in the filename is important to take out because anti-bot operations could look for that

💯 %

File: src/ai_hawk/linkedIn_easy_applier.py [hunk 2]
```diff
@@ -452,7 +452,11 @@ def _handle_upload_fields(self, element: WebElement, job) -> None:
 
         logger.debug("Finished handling upload fields")
 
-    def _create_and_upload_resume(self, element, job):
+    def _create_resume_filepath(self,job:Job,folder_path:str,version:int)->str:
+        return os.path.join(folder_path, f"CV_{job.company}({version}).pdf")
```
Comment (surapuramakhil) [inline src/ai_hawk/linkedIn_easy_applier.py:456]: why this function called create_resume when path is CV_

**Comment by sarob** [comment]
I will set aside some time to check this out

**Comment by sarob** [comment]
I am digging into using this code to 

1. update the new job_description json files
2. moving the CV with this new file name format to the job_description location

**Comment by feder-cr** [comment]
@sarob @FrancescoVassalli hi, any updates about this?

**Comment by feder-cr** [comment]
@FrancescoVassalli ?

**Pull request closed without merging**